src/UI/components/upload/validation/validationConsole.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { render } from '../../../test_config/render';
+import { screen } from '@testing-library/dom';
+import '@testing-library/jest-dom';
+import ValidationConsole from './validationConsole';
+import { AppState } from '../../../state/store';
+import { initialState } from '../../../state/upload/uploadSlice';
+import { useAppSelector } from '../../../state/hooks';
+
+jest.mock('../../../state/hooks', () => ({
+  useAppSelector: jest
```

Comment:
Again, we shouldn't have to mock this - see other comment. At least it's being used in this component...